wiki/getting-started/Configuration.md

```diff
@@ -28,7 +28,10 @@ The name of those config-files (without the `.conf`-extension) will be the ident
 identifier in your commands to select the map. If you rename a config-file, it will be like creating a new map, since bluemap
 also stores all map-information based on that identifier.
 
-If you want to remove a map, you can just delete it's `.conf` file.
+If you want to remove a map, you can just delete its `.conf` file.  
+After removing the file, and reloading/restarting BlueMap, you may want to delete any of the map's previously rendered map data, which you can do with `/bluemap storages <storage> delete <map>`.  
+You can also do that manually by deleting the folder for it, which usually is at `bluemap/web/maps/`, or by deleting the SQL Table for it.
```

Comment:
```suggestion
On the default file-storage, you can also do that manually by deleting the folder for it, which usually is at `bluemap/web/maps/`.
```

On SQL there is not a separate table per map, and there is generally more to consider when deleting a map manually :)

Alternatively just remove the entire sentence as the command should be good enough, manual deletion should not be needed in a normal case. That's why I added the command after all ^^

Reply:
For people using the CLI, it could still be useful to mention, as they don't have access to commands.

Reply:
I'll leave people who are using SQL to deal with it themselves. Their mistake, anyway, to use it  :smiling_imp: